frontend/src/metabase/query_builder/containers/QueryBuilder.jsx

```diff
@@ -354,19 +364,62 @@ function QueryBuilder(props) {
     }
   });
 
+  const { isRunning } = uiControls;
+
+  const [sendNotification, setSendNotification] = useState(false);
+  const [showToaster, setShowToaster] = useState(false);
```

Comment:
Should we maybe rename `sendNotification` and `showToaster` to indicate they're actually booleans? Names starting with a verb look like functions